src/plugins/condition/components/ConditionSet.vue

```diff
@@ -60,11 +62,6 @@ export default {
     mounted() {
         this.conditionSetIdentifier = this.openmct.objects.makeKeyString(this.domainObject.identifier);
     },
-    beforeDestroy() {
```

Comment:
Why is this not needed?

Reply:
Because we're not subscribing to telemetry anymore.